test/test-trace-agent.js

```diff
@@ -160,6 +160,16 @@ describe('Trace Agent', function() {
         });
       });
     });
+
+    describe('when configured to ignore header', function() {
+      it('should return expected value: null', function() {
+        var agent2 = agent;
+        agent2.config_.ignoreContextHeader = true;
+        var result = agent2.parseContextFromHeader(
+          '123456/667;o=1');
+        assert(!result);
```

Comment:
Could the value of `agent.config_. ignoreContextHeader` be reset to the default value `false` here so that it does not remain set for tests run after this one?